### DISCUSSION TITLE
java app with laragon

### DISCUSSION THREAD

**Discussion by edisnp** [leokhoa/laragon#785] [Q&A]
There is no any description on the official website about how to run java server app with laragon...?

**Comment by Pen-y-Fan** [comment]
Try **Laragon menu >Tools > Quick add > tomcat**

![image](https://github.com/leokhoa/laragon/assets/40126936/b7ba828f-646b-4f1a-90e5-3a31320d612c)

I've never used it myself, let us know how you get on.

**Reply by edisnp** [reply]
It seems laragon doesn't support tomcat...
